spec/lib/permission_checker_spec.rb

```diff
@@ -8,6 +8,7 @@
     it { is_expected.not_to have_permission(:user_can_administer_taxonomy?) }
     it { is_expected.not_to have_permission(:user_can_manage_taxonomy?) }
     it { is_expected.not_to have_permission(:user_can_access_tagathon_tools?) }
+    it { is_expected.not_to have_permission(:user_can_administer_facet_groups?) }
```

Comment:
Nitpick: Can you add this expectation to the "Managing Editor" and "Tagathon Participant" tests as well?

Reply:
ah, yes!